.github/workflows/build.yml

```diff
@@ -34,14 +34,9 @@ jobs:
       - name: Validate composer.json and composer.lock
         run: composer validate
 
-      - name: Install dependencies for PHP 7
-        if: matrix.php-versions < '8.0'
+      - name: Install dependencies for PHP
         run: composer update --prefer-dist --no-progress
 
-      - name: Install dependencies for PHP 8
-        if: matrix.php-versions >= '8.0'
-        run: composer update --prefer-dist --no-progress --ignore-platform-reqs
```

Comment:
Finally, I found current solution about the `composer update` commands.

And it's referred the [build.yml](https://github.com/selective-php/archive-bomb-scanner/blob/master/.github/workflows/build.yml) file